cli/tests/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#   Copyright 2011-2015 Quickstep Technologies LLC.
+#   Copyright 2015-2016 Pivotal Software, Inc.
```

Comment:
Since this is a new file, you don't need these two copyright from `Quickstep Technologies LLC` and `Pivotal`. Instead, add the university one. Thanks!